src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.search--bar {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.bars--big {
+  height: 70px;
+}
+
+.bars--small {
+  height: 42px;
+}
+
+.search--bars {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+}
+
+.search--bars:hover {
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.images__big {
```

Comment:
it's better to make this image absolute to make shadow fills a full form
<img width="506" alt="Screenshot 2023-11-06 at 13 16 06" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/117587463/52b281b6-de31-48f4-8a5f-0bc254f0a0c9">